src/pages/STG_PanelTDTM.page

```diff
@@ -1,96 +1,88 @@
-            </apex:outputPanel>
             </apex:outputPanel>
- 
-            <apex:pageBlockTable value="{!listTH}" var="th" >
-               <!-- <apex:column headerValue="{!$Label.stgLabelActionColumn}" styleClass="actionColumn" style="text-align:center;" >
-                   <apex:commandLink action="{!deleteTh}" 
-                       title="{!$Label.stgLinkDelete} {!th.Object__c} - {!th.Class__c}"                                                    
-                       styleClass="actionLink" status="statusLoad" value="{!$Label.stgLinkDelete}" 
-                       onclick="if (confirm('{!$Label.stgLabelAreYouSure}')==false) { return false;}" rerender="form" >
-                       <apex:param assignTo="{!thIdDelete}" name="idDeleteTh" value="{!th.Id}" /> 
-                   </apex:commandLink>
-               </apex:column>  -->
-               <apex:column value="{!th.Object__c}" />
-               <apex:column value="{!th.Class__c}" />
-               <apex:column value="{!th.Load_Order__c}" />
-               <apex:column value="{!th.Trigger_Action__c}" />
-               <apex:column value="{!th.Active__c}" />
-               <apex:column value="{!th.Asynchronous__c}" />
-               <apex:column value="{!th.User_Managed__c}" />
-           </apex:pageBlockTable>
-        </apex:pageBlock> 
+            <table class="slds-table slds-table--bordered" >
+                <thead>
+                    <tr class="slds-text-heading--label">
+                        <th scope="col"><label class="slds-truncate">{!$ObjectType.Trigger_Handler__c.Fields.Object__c.Label}</label></th>
+                        <th scope="col"><label class="slds-truncate">{!$ObjectType.Trigger_Handler__c.Fields.Class__c.Label}</label></th>
+                        <th scope="col"><label class="slds-truncate">{!$ObjectType.Trigger_Handler__c.Fields.Load_Order__c.Label}</label></th>
+                        <th scope="col"><label class="slds-truncate">{!$ObjectType.Trigger_Handler__c.Fields.Trigger_Action__c.Label}</label></th>
+                        <th scope="col"><label class="slds-truncate">{!$ObjectType.Trigger_Handler__c.Fields.Active__c.Label}</label></th>
+                        <th scope="col"><label class="slds-truncate">{!$ObjectType.Trigger_Handler__c.Fields.Asynchronous__c.Label}</label></th>
+                        <th scope="col"><label class="slds-truncate">{!$ObjectType.Trigger_Handler__c.Fields.Load_Order__c.Label}</label></th>
+                        <th scope="col"><label class="slds-truncate">{!$ObjectType.Trigger_Handler__c.Fields.User_Managed__c.Label}</label></th>
+                    </tr>
+                </thead>
+                <tbody>
+                    <apex:repeat var="th" value="{!listTH}" >
+                        <tr>
+                            <td><div class="slds-text-body--small">{!th.Object__c}</div></td>
```

Comment:
this table becomes quite wide.  should we consider using xxsmall text size instead?